horovod/common/operations.cc

```diff
@@ -1644,16 +1500,19 @@ bool RunLoopOnce(HorovodGlobalState& state, MPIContext& ctx,
     }
     RequestList::SerializeToString(message_list, encoded_message);
```

Comment:
Corollary to above, here we're sending the ready tensors, so: `SendReadyTensors()`

Reply:
Done.